website/docs/reference/templating.mdx

```diff
@@ -375,6 +375,7 @@ Lastly, Task itself provides a few functions:
 | ------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | `OS`         | Returns the operating system. Possible values are `windows`, `linux`, `darwin` (macOS) and `freebsd`.                                                                                                  |
 | `ARCH`       | return the architecture Task was compiled to: `386`, `amd64`, `arm` or `s390x`.                                                                                                                        |
```

Comment:
If you're feeling kind, you could update the casing here too. Now that I've spotted this, I can't unsee it 😆 

```suggestion
| `ARCH`       | Return the architecture Task was compiled to: `386`, `amd64`, `arm` or `s390x`.                                                                                                                        |
```

Reply:
Done. Added an `s` as well😀